scripts/run_all_benchmarking.sh

```diff
@@ -22,7 +22,9 @@
 # Should be run on a reference machine to gain accurate benchmarks
 # current reference machine: https://github.com/paritytech/substrate/pull/5848
 
-while getopts 'bfp:v' flag; do
+dev_mode=false
```

Comment:
this line can be omitted, `dev_mode` is `false` by default, as `skip_build` flag